.github/workflows/build-docker-image.yml

```
@@ -71,15 +71,16 @@ jobs:
           # list of Docker images to use as base name for tags
           images: |
             us-docker.pkg.dev/zealous-zebra/zebra/${{ inputs.image_name }}
+            zfnd/zebra,enable=${{ github.event_name == 'release' && !github.event.release.prerelease }}
           # generate Docker tags based on the following events/attributes
           tags: |
             type=schedule
-            type=sha
-            type=ref,event=branch
-            type=ref,event=pr
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
+            type=semver,pattern=v{{major}}
```

Is it going to confuse users if some version tags start with `v`, and others don't?
```
  zfnd/zebra:1.0.0
  zfnd/zebra:1.0
  zfnd/zebra:v1
```

```suggestion
            type=semver,pattern={{major}}
```